data-default-class/data-default-class.cabal

```diff
@@ -1,6 +1,6 @@
 Name:            data-default-class
-Version:         0.1.0
-Cabal-Version:   >= 1.6
+Version:         0.1.0.1
```

Comment:
First of all, 0.2.0 would mean a major version bump, however, if you talk about backward compatible API additions, then the PVP only requires merely a minor version bump. So `0.1.1` would be it.

However, the patch-level increment was under the assumption that 0.1.0 would get a tighter upper bound and so you woulnd't be able to observe the difference between 0.1.0 and 0.1.0.1;  the whole point of this PR is to provide a  non-observable upgrade path.

I'm fully aware this is all subtly complicated